src/stores/database.js

```diff
@@ -94,6 +94,7 @@ class Database {
     const allFilesInMigrationDirectory = await pGlob(path.resolve(migrationsDirectory, './*.js'));
     const migrationFileNames = allFilesInMigrationDirectory
       .filter(fileName => MIGRATION_FILE_NAME_PATTERN.test(path.basename(fileName)))
+      .filter(filename => !filename.includes('manually'))
```

Comment:
Why not put the whole suffix here? You can say `!filename.endsWith('-manually-run')`

Reply:
Yeah, that would be much safer

Reply:
I considered this safer since you rely on manually and not -manually-run (which is more restrictive). I'll change it though

Reply:
For me it makes no sense set a `-manually-run` postfix and have a rule that doesn't match.
And I think in this situation it's better to accidentally run a script that's made for manual run (which might simply run too long) than to skip running it unintentionally and create DB inconsistency.
Our manually run scripts should anyway be re-runnable.

Reply:
Done

Reply:
Pssst, please make it consistent with the rule we agreed on, `!filename.endsWith('-manually-run')`, I already mentioned it here https://github.com/educandu/educandu/pull/35#discussion_r752381844 🙏

Reply:
Aye aye, done